Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_description.md

```diff
@@ -10,7 +10,4 @@
 
 ### URL
 1. In your Cortex XDR platform, go to **Settings**.
-2. Click the **Copy URL** button in the top right corner.
-
----
-[View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/cortex-xdr---ir)
```

Comment:
Is there a reason to remove this? please revert.

Reply:
Was not removed by me.  Possible issue during marketplace contribution process.

Reply:
Ok I'll check it and revert this :)